/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/pytorch_lightning/loggers/wandb.py:396: UserWarning: There is a wandb run already in progress and newly created instances of `WandbLogger` will reuse this run. If this is not desired, call `wandb.finish()` before instantiating `WandbLogger`.
  rank_zero_warn(
Global seed set to 23
Traceback (most recent call last):
  File "main.py", line 225, in <module>
    main_triplet(wandb, wandb_logger)
  File "main.py", line 175, in main_triplet
    data.prepare_data()
  File "/Users/emilywesel/Documents/GitHub/NCANDA_Contrastive/multimodal_dataset_triplet.py", line 230, in prepare_data
    train_subj, test_subj, y_train, y_test, age_train, age_test, group_by_construct_train, group_by_construct_test = self.get_stratified_split(CSV_FILE)
  File "/Users/emilywesel/Documents/GitHub/NCANDA_Contrastive/multimodal_dataset_triplet.py", line 163, in get_stratified_split
    csv_df = pd.read_csv(csv_file)
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/pandas/util/_decorators.py", line 331, in wrapper
    return func(*args, **kwargs)
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1735, in _make_engine
    self.handles = get_handle(
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/pandas/io/common.py", line 856, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/scratch/users/ewesel/NCANDA/Tabular/full_per_visit_data_2021-03-26_processed cross_sectional_scratch.csv'